addons/pos_restaurant/static/src/xml/Screens/PaymentScreen/PaymentScreenElectronicPayment.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="PaymentScreenElectronicPayment" t-inherit="point_of_sale.PaymentScreenElectronicPayment" t-inherit-mode="extension" owl="1">
+        <xpath expr="//div[hasclass('send_payment_reversal')]/.." position="replace">
+            <t t-if="props.line.canBeAdjusted() &amp;&amp; props.line.order.get_total_paid() &lt; props.line.order.get_total_with_tax()">
```

Comment:
In owl templates, we can use `and` instead of `&amp;&amp;` and `lt` instead of `&lt;`. But that is just personal preference.

Reply:
Nice, I didn't know that.